### ISSUE TITLE
[$15][MSFT-33] [Challenge Feed] User can't select the 'Filters' using tab

### ISSUE THREAD

**Issue by veshu** [topcoder-platform/community-app#2755]
**Steps to Reproduce**
1. Go to https://www.topcoder.com/challenges and Login using valid credentials
2. Press Tab key till filter is selected

**Expected Result**
User should be able to select the filter by using the tab key

**Actual Result**
User can't select the 'Filters' using tab Key

**Device**: Desktop/Labtop
**Operating System**: Windows
**Browser**: Chrome

**WCAG Category**: 2. Operable
**WCAG Level**: A
**WCAG Success Criterion**: 2.1.1 Keyboard
**Screen Reader Used**: N/A
**Accessibility Audit Tool used**: Other

**Attachment**
https://topcodermsft-my.sharepoint.com/personal/pd-topcoder_topcodermsft_onmicrosoft_com/Documents/Forms/All.aspx?cid=58da9b68%2Df2a3%2D45d2%2Dbbae%2D15abdfd0865b&FolderCTID=0x0120005C598A51049FC14CBA882E1AEE168F51&sortField=Modified&isAscending=true&id=%2Fpersonal%2Fpd%2Dtopcoder%5Ftopcodermsft%5Fonmicrosoft%5Fcom%2FDocuments%2FMSFT%2DTeams%2DQA%2FJune%202019%2F29%20Jun%2FTest%20Results%2FBug%20Videos%20%26%20Log%20Files%2Fcodejam%2DAccessibility%2F18

**Comment by ThomasKranitsas** [comment]
[400]: Failed to create challenge. Detail: Operation failed in the contest service facade.<br/><br/>```This is an automated message for thomaskranitsas via Topcoder X```

**Comment by ThomasKranitsas** [comment]
[400]: Failed to create challenge. Detail: Operation failed in the contest service facade.<br/><br/>```This is an automated message for thomaskranitsas via Topcoder X```

**Comment by ThomasKranitsas** [comment]
Contest https://www.topcoder.com/challenges/30096156 has been created for this ticket.<br/><br/>```This is an automated message for thomaskranitsas via Topcoder X```

**Comment by r0hit-gupta** [comment]
Please assign me

**Comment by Oanh-and-only-Oanh** [comment]
@r0hit-gupta, please update your tickets with "tcx_readyforreview label if it's ready. thanks

**Comment by veshu** [comment]
I think he is not able to update the ticket till he comes online I will be making those as ready for review.

**Comment by r0hit-gupta** [comment]
@Oanh-and-only-Oanh @veshu I am unable to assign myself or change labels. What should I do?

**Comment by veshu** [comment]
@r0hit-gupta Can you please register here https://www.topcoder.com/challenges/30095935/?type=develop&nocache=true&tab=details and then use this link to register https://x.topcoder.com/api/v1/github/teams/registration/462d863d-9f1a-4179-a215-29f71a551e4f-1563464121672

**Comment by r0hit-gupta** [comment]
Still receiving `{"version":"v3","result":{"success":false,"status":403,"content":{"message":"Failed to authenticate token."}}}` :(

**Comment by tosha5252** [comment]
...

**Comment by tosha5252** [comment]
failed testing at https://test-community-app.topcoder-dev.com/challenges...

Tab skips over 'Filters' from Search Bar, going down to My Challenges.

**Comment by tosha5252** [comment]
Env:

OS: Windows 10
Browser: Chrome

**Comment by r0hit-gupta** [comment]
@tosha5252 looks like the code isn't deployed properly. I am unable to see the modifications I made in the DOM.

**Comment by veshu** [comment]
@tosha5252 This looks fixed, in live site i can't select the filters button using keyboard, after search box tab goes to challenges while in above site i can go to filters button after search box and pressing enter key I can open the filter options.

cc @lakshmiathreya @drasticdpk

**Comment by lakshmiathreya** [comment]
Fixed in https://test-community-app.topcoder-dev.com/challenges. Verified on MacOS/Chrome and Win 10/Edge (browserstack). Moving to QA Pass.

**Comment by cwdcwd** [comment]
Contest https://www.topcoder.com/challenges/30097516 has been created for this ticket.<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by cwdcwd** [comment]
Contest https://www.topcoder.com/challenges/30097516 has been updated - it has been assigned to rohitgupta_.<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by tosha5252** [comment]
Verified working 

Env: Windows 10
Chrome and Edge

**Comment by crazyk07** [comment]
Payment task has been updated: https://software.topcoder.com/review/actions/ViewProjectDetails?pid=30097516<br/><br/>```This is an automated message for crazyk via Topcoder X```
